src/many/src/main.rs

```diff
@@ -200,7 +200,7 @@ struct MessageOpt {
     data: Option<String>,
 
     #[clap(long)]
-    attribute: Option<u32>,
+    proof: Option<bool>,
```

Comment:
Doesn't have to be `Option<bool>`, just set default value.

Reply:
OK

Reply:
Sorry, I can make a suggestion :) I was going quick through the PR

Reply:
Also noted that there's a missing description for this flag.

```rust
    /// Request a proof of the value. This may cause an error if the server
    /// does not support proofs, and might not work on all endpoints. Consult
    /// the specification for more information.
    #[clap(long, default_value = "false")]
    proof: bool,
```

Reply:
Ah . . . I hadn't done it this way, but will now

Reply:
Done